Keas.Mvc/Models/ServiceNowProperties.cs

```diff
@@ -0,0 +1,65 @@
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Keas.Mvc.Models
+{
+    public class ServiceNowPropertyWrapper
+    {
+        [JsonProperty("result")]
+        public List<ServiceNowProperties> Result { get; set; }
```

Comment:
this should be called results since it's a list

Reply:
Okay I can change that